### ISSUE TITLE
Claim command

### ISSUE THREAD

**Issue by XxRazorxX** [Dragory/modmailbot#174]
i thought of the idea that one moderator can claim a thread, like !claim and then only one moderator will be abled to help someone through this log!

**Comment by Dragory** [comment]
I agree, this would be a great feature to add. I'll add it to the todo.

**Comment by NexExMachina** [comment]
is there any update to this?
